doc/sphinx_source/modules/internals.rst

```diff
@@ -125,23 +146,65 @@ Triggering any Bind
     return x;
   }
 
-The supplied flags to :code:`check_tcl_bind` in `check_tcl_dcc` are what defines how matching is performed.
-
-In the case of a DCC bind we had:
-
-* Matchtype :code:`MATCH_PARTIAL`: Prefix-Matching if the command can be uniquely identified (e.g. dcc .help calls .help)
-* Additional flag :code:`BIND_USE_ATTR`: Flags are checked
-* Additional flag :code:`BIND_HAS_BUILTINS`: Something with flag matching, unsure
-
-For details on the available match types (wildcard matching, exact matching, etc.) see :code:`src/tclegg.h`. Additional flags are also described there as well as the return codes of :code:`check_tcl_bind` (e.g. :code:`BIND_NOMATCH`).
+Bind Flags
+^^^^^^^^^^
+
+The last argument to :code:`check_tcl_bind` in `check_tcl_dcc` sets additional configurations for the bind. These are the allowed defined values:
+
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| **Value**         | **Description**                                                                                                               |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| MATCH_PARTIAL     | Check the triggering value against the beginning of the bind mask, ie DIR triggers a mask for DIRECTORY (case insensitive)    |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| MATCH_EXACT       | Check the triggering value exactly against the bind mask value (case insensitive)                                             |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| MATCH_CASE        | Check the triggering value exactly against the bind mask value (case sensitive)                                               |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| MATCH_MASK        | Check if the bind mask is contained within the triggering value, as a wildcarded value                                        |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| MATCH_MODE        | Check if the triggering value is contained within the bind mask, as a wildcarded value                                        |
++-------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| MATCH_CRON        | Check the triggering value against a bind mask formatted as a cron entry, ie "30 7 6 7 * " triggers a mask for "30 7 * * * "  |
```

Comment:
The example should not contain * anymore, it is triggered with concrete values like "30 7 6 7 5"

Reply:
From the comments in match.c,
```
/* Check if the current time matches a crontab-like specification.
 *
 * mask contains a cron-style series of time fields. The following
 * crontab operators are supported: ranges '-', asterisks '*',
 * lists ',' and steps '/'.
```    
are we sure?